src/sidebar/components/annotation.js

```diff
@@ -1,318 +1,169 @@
-import events from '../events';
-import { isThirdPartyUser } from '../util/account-id';
-import { isNew, isReply, isPageNote, quote } from '../util/annotation-metadata';
+import classnames from 'classnames';
+import { createElement } from 'preact';
+import { useState } from 'preact/hooks';
+import propTypes from 'prop-types';
+
+import useStore from '../store/use-store';
+import { isNew, isReply, quote } from '../util/annotation-metadata';
+import { isShared } from '../util/permissions';
+import { withServices } from '../util/service-context';
+
+import AnnotationActionBar from './annotation-action-bar';
+import AnnotationBody from './annotation-body';
+import AnnotationHeader from './annotation-header';
+import AnnotationLicense from './annotation-license';
+import AnnotationPublishControl from './annotation-publish-control';
+import AnnotationQuote from './annotation-quote';
+import Button from './button';
+import TagEditor from './tag-editor';
+import TagList from './tag-list';
 
 /**
- * Return a copy of `annotation` with changes made in the editor applied.
+ * The "new", migrated-to-preact annotation component.
```

Comment:
This comment needs updating now that there is only one annotation implementation.